tests/e2e/backup_restore_suite_test.go

```diff
@@ -170,7 +165,7 @@ var _ = Describe("AWS backup restore tests", func() {
 			}
 
 		},
-		Entry("MSSQL application CSI", BackupRestoreCase{
+		Entry("MSSQL application CSI", Label("aws"), BackupRestoreCase{
```

Comment:
why this label has been applied only to first Entry?

Reply:
instead of having a condition inside the test logic:
```

			if brCase.BackupRestoreType == csi {
				if clusterProfile == "aws" {
					log.Printf("Creating VolumeSnapshot for CSI backuprestore of %s", brCase.Name)
					err = installApplication(dpaCR.Client, "./sample-applications/gp2-csi/volumeSnapshotClass.yaml")
					Expect(err).ToNot(HaveOccurred())
				} else {
```